sdcm/tombstone_gc_verification_thread.py

```diff
@@ -10,40 +8,26 @@
 from sdcm.remote import LocalCmdRunner
 from sdcm.sct_events import Severity
 from sdcm.sct_events.database import TombstoneGcVerificationEvent
-from sdcm.utils.decorators import retrying
+from sdcm.utils.sstable_utils import SstableUtils
 
 LOCAL_CMD_RUNNER = LocalCmdRunner()
 ERROR_SUBSTRINGS = ("timed out", "timeout")
 
 
-class NonDeletedTombstonesFound(Exception):
-    pass
-
-
-class NoSstableFound(Exception):
-    pass
-
-
 # pylint: disable=too-many-instance-attributes
-class TombstoneGcVerificationThread:
+class TombstoneGcVerificationThread(SstableUtils):
```

Comment:
Why using `SstableUtils` as a base class? It introduces complexity...
Why not create instance of `SstableUtils` inside, like e.g.:
`self._sstable_utils = SstableUtils(....)`

Let's keep single purpose classes so it's easier to test and comprehend.

Reply:
Also very similar in name to

SstableLoadUtils that sits in sdcm.utils.sstable.load